cfg/aws/aws_sdk_logging.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package aws
+
+import (
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/influxdata/wlog"
+)
+
+var sdkLogLevel aws.LogLevelType = aws.LogOff
+
+// SetSDKLogLevel sets the global log level which will be used in all AWS SDK calls.
+// Example usage: set `debug = true` and `aws_sdk_log_level = LogDebug` in config.json.
+func SetSDKLogLevel(wLogLevel wlog.Level, sdkLogLevelString string) {
+	if wLogLevel != wlog.DEBUG {
+		sdkLogLevel = aws.LogOff
+		return
+	}
+	var temp aws.LogLevelType = aws.LogOff
+	// Hard coded strings that match actual variable names in the AWS SDK.
+	// I don't expect these names to change, or their meaning to change.
+	// This code may need updates if the SDK adds more levels (unlikely).
+	// Please note that the levels are a bit field that is OR'd together.
+	if strings.Contains(sdkLogLevelString, "LogDebug") {
```

Comment:
Why `Contains` instead of `==`?

Reply:
I see.  Maybe we could be more strict about the syntax then?  Like split on `|` and match the string exactly?  Or provide an array of values like `["LogDebugWithSigning", "LogDebugWithRequestRetries"]`?  Just makes me nervous that someone could pass `"BogusStringLogDebugBogusString"` and it would still be matched.

Reply:
I have no issue with either.
I will choose the first option and just string split on `|` and look for exact string match.